ordersettler/ordersettler.go

```diff
@@ -218,12 +204,16 @@ func (r *OrderSettler) settleOrders(ctx context.Context) error {
 // verifyOrderSettlements checks on all instated settlements and updates their
 // status in the db with their on chain tx results.
 func (r *OrderSettler) verifyOrderSettlements(ctx context.Context) error {
+	pendingSettlements, err := r.db.GetAllOrderSettlementsWithSettlementStatus(ctx, dbtypes.SettlementStatusPending)
```

Comment:
verifyOrderSettlement is responsible for updating settlement state from pending to initiated. probably worth refactoring the functionality here for more clarity in the future.